pom.xml

```diff
@@ -105,9 +106,9 @@
                just remove the overriding feature and blacklist entry. To use a different version
                of jetty, the overriding feature will need to be replaced and the blacklist entry
                updated to disable karaf's jetty version.-->
-        <karaf.version>4.1.2</karaf.version>
+        <karaf.version>4.2.0</karaf.version>
```

Comment:
❓ Why aren't we upgrading to 4.2.1?

Reply:
4.2.1 came out just a couple of days before I got this PR up. I *did* try to go to 4.2.1 and ran into several issues with itests, so holding off for now.